### PR TITLE
Function instrumentation

### PR DESCRIPTION
**Pull request by Avaquet**
New version of the function instrumentation, fix the bug with intrinsics.

### CODE REVIEW

File: src/libvfcfuncinstrument/libVFCFuncInstrument.cpp
Comment (titeup) [inline src/libvfcfuncinstrument/libVFCFuncInstrument.cpp:215]: You are cloning only the function attributes? From our discussion I understood we need to clone arguments' attributes. Or both. Mabe we can clarify during our next meeting.

Comment (titeup) [reply src/libvfcfuncinstrument/libVFCFuncInstrument.cpp:215]: Hum... thinking twice, since you are using the parameter list of the cloned function you have the same attributes... This copy is only necessary when you duplicate your parameters. and I cannot see it in your previous 'step'

Comment (titeup) [reply src/libvfcfuncinstrument/libVFCFuncInstrument.cpp:215]: We check with allan today in IR, the copy in step2 is indeed copying the attribute of the arguments. Case closed. the rest is good for me. but I prefer to have a second review before merging.

Comment (yohanchatelain) [reply src/libvfcfuncinstrument/libVFCFuncInstrument.cpp:215]: LGTM

**Review by yohanchatelain — APPROVED** [review]

**Pull request merged**
